src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -73,12 +68,19 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
   }
 
   const postMaker = postNotifItem?.rawJson?.postMaker;
-  const isOwnPost = postNotifItem?.rawJson?.isOwnPost;
   const isOwnSignedPost = item?.rawJson?.isOwnSignedPost;
 
   const {determinePostType} = useChannelHook();
   const type = determinePostType(postNotifItem);
 
+  const getIsMe = () => {
+    if (item?.channelType === 'ANON_POST_NOTIFICATION')
+      return postNotifItem?.rawJson?.isOwnAnonymousPost;
+    if (item?.channelType === 'POST_NOTIFICATION') return postNotifItem?.rawJson?.isOwnSignedPost;
+
+    return false;
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of the `getIsMe` function seems to be a good change as it encapsulates the logic for determining whether the current user is the one who made the post. However, please ensure that the `item?.channelType` values 'ANON_POST_NOTIFICATION' and 'POST_NOTIFICATION' are the only possible values. If there are other possible values, you may need to handle those cases as well.
```diff
-    const isOwnPost = postNotifItem?.rawJson?.isOwnPost;
+    const getIsMe = () => {
+      if (item?.channelType === 'ANON_POST_NOTIFICATION')
+        return postNotifItem?.rawJson?.isOwnAnonymousPost;
+      if (item?.channelType === 'POST_NOTIFICATION') return postNotifItem?.rawJson?.isOwnSignedPost;
+
+      return false;
+    };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->